lib/tessel/deployment/lists/javascript.js

```diff
@@ -54,7 +54,14 @@ module.exports = {
     },
   },
   compressionBlacklist: {
-    bindings: true,
-    base64id: true,
```

Comment:
Do we still need this in the blacklist?

Reply:
Yes, it's being used for "supports-color" right now

Reply:
Oh, hrm... I think maybe it doesn't need to?